src/app/blogs/[id]/page.tsx

```diff
@@ -0,0 +1,42 @@
+"use client";
+
+import Card from "react-bootstrap/Card";
+import Button from "react-bootstrap/Button";
```

Comment:
@HoVanDi - pls remove this line due to it's redundant (it's declared but its value never used)